File: 4기 코딩테스트_220615/4기 - 2nd 코딩테스트/IndexOfWord/IndexOfWord.py
def solution(sentence, word):
    """
    :param sentence: string
    :param word: string
    :return: int
    """

    if word == '':
        return -1

    words = sentence.split(' ')

    for i, eachWord in enumerate(words):
        print(word)
        if eachWord == word:
            return i

    return -1

File: 4기 코딩테스트_220615/4기 - 2nd 코딩테스트/IndexOfWord/IndexOfWord.js
/**
 * @param sentence {string}
 * @param word {string}
 * @returns {number}
 */
function solution(sentence, word) {
  if (word === '') return -1

  const words = sentence.split(' ')

  for (let i = 0; i < words.length; i++) {
    if (words[i] === word) return i
  }

  return -1
}

export default solution
